index.html

```diff
@@ -6,6 +6,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>La grève, c’est cool</title>
+
+    <meta property="og:title" content="la grève, c’est cool" />
+    <meta property="og:type" content="website" />
+    <meta property="og:locale" content="website" />
+    <meta property="og:url" content="fr-FR" />
```

Comment:
```suggestion
    <meta property="og:url" content="https://greve.cool/" />
```

Je pense qu'il y a une erreur sur ce champ. Avec une URL, c'est mieux :)

Reply:
Nan je me suis chié dessus je voulais mettre locale.

Reply:
Enfin j'ai mélangé les 3 champs comme un bolosse.